### PR TITLE
Update device_tracker.unifi.markdown

### PR DESCRIPTION
**Pull request by rbflurry**
**Description:**
For 
https://github.com/home-assistant/home-assistant/pull/8998

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8998

### CODE REVIEW

**Comment by ravor-org** [comment]
You forgot about the api-parameter... :)

**Comment by rbflurry** [comment]
@ravor-org Added.

File: source/_components/device_tracker.unifi.markdown [hunk 1]
```diff
@@ -33,5 +33,7 @@ Configuration variables:
 - **password** (*Required*): The password for your given admin account.
 - **site_id** (*Optional*): Allows you to specify a `site_id` for device tracking. Defaults to `default`. Found in the URL of the controller (i.e. https://CONTROLLER:PORT/manage/site/SITE_ID/dashboard).
 - **verify_ssl** (*Optional*): Controls if the SSL certificate running on your Unifi webserver must be trusted by a known Certificate Authority on the server running Home Assistant. Defaults to 'True' but can also be a value that points to your custom cert "path/to/custom_cert.pem".
+- **detection_time** (*Optional*): The unifi component will not return a device that has not been seen by the controller in the last 180 seconds. You can adjust this threshold with this varible. Will except seconds or 00:00:00 time formats.
```
Comment (arsaboo) [inline source/_components/device_tracker.unifi.markdown]: Except -> accept

Comment (arsaboo) [reply source/_components/device_tracker.unifi.markdown]: Capitalize Unifi

**Comment by frenck** [comment]
@rbflurry Looks like the API parameter was removed again in a later iteration of the parent PR.
Please remove it from the documentation as well.

**Comment by frenck** [comment]
Removed API reference and made some spelling & grammar corrections.
Merging it, since parent PR is already merged.

**Pull request merged**